src/Containers/App/App.js

```diff
@@ -1,25 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Provider }
```

Comment:
Het @benjamin-firth , what is this provider for and where is it coming from?

Reply:
Good catch! I was adding that to the index.js file and accidentally put it there.  I thought I'd removed it thanks!